classes/Product.php

```diff
@@ -212,7 +212,7 @@ class ProductCore extends ObjectModel
      *
      * @see RedirectType
      */
-    public $redirect_type = RedirectType::TYPE_NOT_FOUND;
+    public $redirect_type = RedirectType::TYPE_DEFAULT;
```

Comment:
Looks like a huge BC break here :/

Reply:
not as huge as it looks (or not a BC at all?). because the TYPE_DEFAULT will contain the same value as it was under "TYPE_NOT_FOUND" by default in configuration as you can see [here](https://github.com/PrestaShop/PrestaShop/pull/29720/files#diff-d9529f3253ec6cb566b81d6c6a5943efab9fbdd1db543b99ca4cdbbc9995dfdfR895-R896).

Reply:
@FabienPapet What do you have in mind? This won't break anything. No value was deleted from RedirectType class. Also, whatever value there willl be in database, it will fallback both in frontoffice and backoffice.

Reply:
Discussed with Jo and Krystain, no BC break ;-)